test/framework/yeb.py

```diff
@@ -117,6 +117,24 @@ def test_join(self):
         for key in ['fb1', 'fb2', 'fb3']:
             self.assertEqual(loaded.get(key), 'foobar')
 
+    def test_majmin(self):
+        """ Test yaml_versionmajorminor function """
+        stream = [
+            "variables:",
+            "   - &version 2.7.65",
+            "   - &comp_version 1.4.8.7",
+            "",
+            "mm1: !majmin [*version]",
+            "mm2: !majmin [*comp_version]",
```

Comment:
there must be a way to avoid the need for passing a sequence if an alias like `*version*` is used

can you include an extra test case with a hardcoded version, like:

```
mm3: !majmin 1.23.456
```

does that work as expected (without requiring a list)? You'll need to update the function supporting this to differentiate between a string and list value...